<?xml version="1.0"?>
<launch>

  <node pkg="osm_cartography" type="osm_server" name="osm_server"/>
  <node pkg="map_module" type="map_server_node" name="map_server_node" output="screen"/>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find map_module)/rviz/map.rviz" respawn ="true"/>

</launch>
